src/openrct2/actions/MazeSetTrackAction.hpp

```diff
@@ -94,7 +94,7 @@ DEFINE_GAME_ACTION(MazeSetTrackAction, GAME_COMMAND_SET_MAZE_TRACK, GameActionRe
             return res;
         }
 
-        if (!map_is_location_owned(_loc) && !gCheatsSandboxMode)
+        if (!map_is_location_owned({ _loc.x, _loc.y, _loc.z }) && !gCheatsSandboxMode)
```

Comment:
Wasn't the plan to allow decaying CoordsXYZD into CoordsXYZ and likewise?

Reply:
Yes. I haven't got around to doing that though

Reply:
Do you plan to tackle that in this PR or in another?

Reply:
Can't decide. I'll run a few tests this weekend